src/main/java/com/greeenai/greeenai/domain/diary/service/DiaryService.java

```diff
@@ -105,15 +131,40 @@ private void validateDiaryOwner(Long currentMemberId, Long diaryOwnerId) {
         }
     }
 
-    @Transactional(readOnly = true)
-    public List<DiaryResponse> findAllMyDiaries(@Nullable LocalDate entryDate) {
+    private String getDiaryImageUrl(Diary diary) {
+        return imageService.getUrl(diary.getImage());
+    }
+
+    private List<Image> saveUserImages(List<MultipartFile> userImages) {
         Member currentMember = memberUtil.getCurrentMember();
-        List<Diary> myDiaries = diaryRepository.findAllByMemberAndEntryDate(currentMember, entryDate);
+        return userImages.stream()
+                .map(userImage -> imageService.uploadImage(userImage, ImageType.USER, currentMember.getId()))
```

Comment:
```suggestion
                .map(userImage -> imageService.uploadImage(userImage, USER, currentMember.getId()))
```

static import 하는게 좋을 것 같아요~